### PR TITLE
feat(FOROME-1556): redesign enum panel

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1556
And related tasks [task_1](https://quantori.atlassian.net/browse/FOROME-1557), [task_2](https://quantori.atlassian.net/browse/FOROME-1559)

### CODE REVIEW

File: src/components/units-list/unit-chart/bar-chart/bar-chart.tsx [hunk 1]
```diff
@@ -12,32 +13,59 @@ interface IBarChartProps {
...
+    return data.some(item => selectedVariants?.includes(item[0]))
+  }, [data, selectedVariants])
+
+  const renderBarChartLegend = () => {
+    const defaultLegend = t('filter.chart.shownSignificantItems', {
+      items: data.length,
+      total: totalItems,
+    })
+
+    const specialLegent = t('filter.chart.noVisualRepresentation')
```
Comment (EvgeniySobolev) [inline src/components/units-list/unit-chart/bar-chart/bar-chart.tsx]: `specialLegend`, typo

File: src/components/units-list/unit-chart/pie-chart/pie-chart.utils.ts [hunk 2]
```diff
@@ -17,27 +19,19 @@ export const getShortNumber = (value: number): string => {
...
   data,
   width,
   height,
   tooltip,
+  isDashboard,
+  selectedVariants,
+  onSelectVariantByChart,
 }: SvgChartRenderParams<TPieChartData>): void => {
-  const radius = Math.min(width, height) / 2
+  const radius = Math.min(width, height) / 2.02
```
Comment (EvgeniySobolev) [inline src/components/units-list/unit-chart/pie-chart/pie-chart.utils.ts:34]: Why 2.02?

**Review by EvgeniySobolev — APPROVED** [review]

File: src/components/units-list/unit-chart/pie-chart/pie-chart.utils.ts (hunk 2 above)
Comment (Lapk1n) [reply src/components/units-list/unit-chart/pie-chart/pie-chart.utils.ts:34]: Screenshot 2022-08-22 at 15 54 04
To get rid of this cutting

File: src/components/conditions/enum/enum-condition.interface.ts [hunk 3]
```diff
@@ -19,6 +21,9 @@ export interface IEnumConditionProps {
   isDataReady: boolean
   listHeight: string
   selectedDashboardVariants?: string[]
+  selectedAttributeStatus?: TPropertyStatus | undefined
```
Comment (QSdmitrioul) [inline src/components/conditions/enum/enum-condition.interface.ts]: you don't need undefiend

File: src/components/svg-chart/svg-chart.tsx [hunk 4]
```diff
@@ -15,22 +15,31 @@ export type SvgChartRenderParams<Data> = {
...
+  isDashboard?: boolean
+  onSelectVariantByChart?: ((variant: string) => void) | undefined
 }
 
 interface ISvgChartProps<Data> extends SVGProps<SVGSVGElement> {
   width?: number
   height?: number
   component?: 'svg' | ComponentType<SVGProps<SVGSVGElement>>
   data: Data
+  selectedVariants?: string[] | undefined
```
Comment (QSdmitrioul) [inline src/components/svg-chart/svg-chart.tsx]: useless undefined

**Review by QSdmitrioul — APPROVED** [review]

**Pull request merged**